x-pack/plugins/maps/public/actions/layer_actions.ts

```diff
@@ -580,3 +581,27 @@ export function setAreTilesLoaded(layerId: string, areTilesLoaded: boolean) {
     newValue: areTilesLoaded,
   };
 }
+
+export function updateCounts(layerId: string, mbMetaFeatures: Feature[]) {
+  return async (
+    dispatch: ThunkDispatch<MapStoreState, void, AnyAction>,
+    getState: () => MapStoreState
+  ) => {
+    if (!mbMetaFeatures) {
```

Comment:
Is this check needed? The type says that mbMetaFeatures will always be provided. Either change the parameter type or remove this check.